TestCases/incomp_rans/multicomponentflow_variablefluidproperties/c6_cms_venturi_conference.cfg

```diff
@@ -0,0 +1,383 @@
+%DIFFUSIVITY_CONSTANT = 1.0e-4
+DIFFUSIVITY_MODEL = UNITY_LEWIS
+
+MUSCL_SCALAR= YES
+
+SLOPE_LIMITER_SCALAR = NONE
+
+TIME_DISCRE_SCALAR= EULER_IMPLICIT
+
+% SCALAR CLIPPING
+SCALAR_CLIPPING= YES
+SCALAR_INIT = 1
+SCALAR_CLIPPING_MIN= 0.0
+SCALAR_CLIPPING_MAX= 1.0
+
+
+% -------------------- TURBULENT TRANSPORT ---------------------------------------%
+CONV_NUM_METHOD_TURB= SCALAR_UPWIND
+MUSCL_TURB= NO
+%FREESTREAM_TURBULENCEINTENSITY= 0.02
+%FREESTREAM_TURB2LAMVISCRATIO= 5.0
+
+% --------------------------- VISCOSITY MODEL ---------------------------------%
+
+% Viscosity model (SUTHERLAND, CONSTANT_VISCOSITY).
+VISCOSITY_MODEL= CONSTANT_VISCOSITY
+
+% Molecular Viscosity that would be constant (1.716E-5 by default)
+MU_CONSTANT= 1.1102e-05, 1.8551e-05 
+
+% Sutherland Viscosity Ref (1.716E-5 default value for AIR SI)
+MU_REF= 1.118e-05, 1.795e-05
+
+% Sutherland Temperature Ref (273.15 K default value for AIR SI)
+MU_T_REF= 300, 300
+
+% Sutherland constant (110.4 default value for AIR SI)
+SUTHERLAND_CONSTANT= 187.78, 129.4
+
+% Temperature polynomial coefficients (up to quartic) for viscosity.
+% Format -> Mu(T) : b0 + b1*T + b2*T^2 + b3*T^3 + b4*T^4
+%MU_POLYCOEFFS= (0.1, 0.1, 0.1, 0.1, 0.1)
+
+% ---------------------- REFERENCE VALUE DEFINITION ---------------------------%
+
+% Reference origin for moment computation
+REF_ORIGIN_MOMENT_X = 0.25
+REF_ORIGIN_MOMENT_Y = 0.00
+REF_ORIGIN_MOMENT_Z = 0.00
+
+% Reference length for pitching, rolling, and yawing non-dimensional moment
+REF_LENGTH= 1.0
+
+% Reference area for force coefficients (0 implies automatic calculation)
+REF_AREA= 1.0
+
+% -------------------- BOUNDARY CONDITION DEFINITION --------------------------%
+
+% Navier-Stokes wall boundary marker(s) (NONE = no marker)
+
+MARKER_HEATFLUX= ( wall, 0.0 )
+MARKER_SYM= ( axis )
+AXISYMMETRIC= NO
+
+INC_INLET_TYPE=  VELOCITY_INLET VELOCITY_INLET VELOCITY_INLET
+
+% Inlet boundary type 
+MARKER_INLET_SCALAR = (gas_inlet, 1.0, air_axial_inlet, 0.0, air_radial_inlet, 0.0)
+MARKER_INLET= ( gas_inlet, 300, 12.353484, 1.0, 0.0, 0.0, air_axial_inlet, 300, 1.7857917, 1.0, 0.0, 0.0, air_radial_inlet, 300, 6.9766378, 0.0, -1.0, 0.0 )
+
+INC_OUTLET_TYPE= PRESSURE_OUTLET
+
+MARKER_OUTLET= ( outlet, 0.0)
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+
+% Numerical method for spatial gradients (GREEN_GAUSS, WEIGHTED_LEAST_SQUARES)
+%NUM_METHOD_GRAD= GREEN_GAUSS
+NUM_METHOD_GRAD= WEIGHTED_LEAST_SQUARES
+
+% Courant-Friedrichs-Lewy condition of the finest grid
+
+CFL_NUMBER = 60
+
+CFL_REDUCTION_SCALAR = 1.0
+CFL_REDUCTION_TURB = 1.0
+%RELAXATION_FACTOR_FLOW   = 1
+%RELAXATION_FACTOR_SCALAR = 1
+
+% Adaptive CFL number (NO, YES)
+CFL_ADAPT= NO
+
+% Parameters of the adaptive CFL number (factor down, factor up, CFL min value,
+%                                        CFL max value )
+CFL_ADAPT_PARAM= ( 1.5, 0.5, 1.0, 100.0 )
+
+
+% Runge-Kutta alpha coefficients
+RK_ALPHA_COEFF= ( 0.66667, 0.66667, 1.000000 )
```

Comment:
```suggestion
```
Remove redundant options from the config.

Reply:
Will do 👍